packages/admin-panel/src/pages/resources/ResourcePage.js

```diff
@@ -84,6 +87,7 @@ ResourcePage.propTypes = {
 };
 
 ResourcePage.defaultProps = {
+  ConfirmDeleteModalComponent: undefined,
```

Comment:
Needs to be undefined as opposed to null so that the DataFetchingModal component uses the default if nothing is passed in here.